cypress/integration/4.quickedit/create.selection.js

```diff
@@ -0,0 +1,62 @@
+// TODO: why are we getting an error for `the_records`
+// here in the course settings page?
+Cypress.on('uncaught:exception', (err, runnable) => {
+    // returning false here prevents Cypress from
+    // failing the test
+    return false;
+});
+
+describe('Instructor creates a selection', () => {
+    before(() => {
+        cy.login('instructor_one', 'test');
+        cy.visit('/course/1/');
+        cy.wait();
```

Comment:
One thing you could try here is chaining `should` to `visit`, like:
```
cy.visit('/course/1/').should(() => { expect(...)});
```
The `expect` syntax comes from Chai, but Cypress has some examples here: https://docs.cypress.io/guides/references/assertions.html#Chai
In any case, the thing that helped me understand this better was that if some action had side effects on the page, like an AJAX request, then use this chained syntax.

Reply:
Here's another example from the docs: https://docs.cypress.io/guides/core-concepts/introduction-to-cypress.html#Asserting-in-English
```
cy.request('/users/1').its('body').should('deep.eq', { name: 'Jane' })
```